_dprhtml/js/dpr_pal.js

```diff
@@ -125,7 +125,7 @@
       });
   }
 
-  DPR_PAL.getDifId = () => /analysis=[^&]/.test(window.location.href) ? 'difb-bottom' : 'difb';
+  DPR_PAL.getDifId = () => DPR_PAL.isNavigationFeature() ? 'difb-bottom' : 'difb';
```

Comment:
It is possible to open the bottom pane via the `1` shortcut and click on a DPPN link before analyzing a word. As a result, checking the URL for the `analysis` parameter is insufficient to determine the correct `difId`.